### DISCUSSION TITLE
Some security about SMS api

### DISCUSSION THREAD

**Discussion by spalekg** [ltb-project/self-service-password#473] [Ideas]
There should be some security about sending SMS.
Someone can guess usernames and flood the owner's mobile with SMS messages, and even that should spend SMS credits for service using for that purpose. Maybe to put some captcha for SMS sending and/or minimum time counter for the next SMS message to the same user.

**Comment by coudot** [comment]
There is already a captcha in SSP, but we need some work to improve it:
* https://github.com/ltb-project/self-service-password/issues/311
* https://github.com/ltb-project/self-service-password/issues/343

Any help is welcomed
